include/gnc/constants.hpp

```diff
@@ -60,7 +60,7 @@ GNC_TRACKED_CONSTANT(constexpr static double, earth_perihelion_time, -6287982.0)
 
 GNC_TRACKED_CONSTANT(constexpr static double, earth_period, 365.256363004 * 24.0 * 60.0 * 60.0);
 
-GNC_TRACKED_CONSTANT(constexpr static lin::Vector4d, q_eci_perifocal, 0.127456632677880, -0.158124280715206, 0.762378784011859, -0.614434787689722);
+GNC_TRACKED_CONSTANT(constexpr static lin::Vector4d, q_eci_perifocal, 0.127456632677880, -0.158124280715206, 0.762378784011859, 0.614434787689722);
```

Comment:
Why is this changed and does it need to be changed in MATLAB?

Reply:
So at some point this got changed in MATLAB but wasn't updated in the C++. As a result, our sun vector environment function was just wrong. Stewart and I figured this out while debugging the filter.

Reply:
Reverted these changes and fixed the attitude filter testcase accordingly.